packages/core/ios/Sources/DdSdkNativeInitialization.swift

```diff
@@ -38,7 +38,7 @@ public class DdSdkNativeInitialization: NSObject {
         self.jsonFileReader = jsonFileReader
     }
     
-    internal func initialize(sdkConfiguration: DdSdkConfiguration) {
+    internal func initialize(sdkConfiguration: DdSdkConfiguration, callback: @escaping () -> Void) {
         // Datadog SDK init needs to happen on the main thread: https://github.com/DataDog/dd-sdk-reactnative/issues/198
         self.mainDispatchQueue.async {
```

Comment:
Initialisation on the main thread is no longer a requirement since `2.0`, you can do the init in sync so no need to have a callback :)

Reply:
ah nice!